### PR TITLE
bench mobly snippets

### PR DESCRIPTION
**Pull request by barbibulle**
Adds mobly snippets for the Bench Android app so that it can be automated.

### CODE REVIEW

File: docs/mkdocs/src/apps_and_tools/bench.md [hunk 1]
```diff
@@ -82,7 +94,7 @@ Device 1 mode     | Device 2 mode
 
 Device 1 role | Device 2 role
```
Comment (AlanRosenthal) [inline docs/mkdocs/src/apps_and_tools/bench.md:95]: I believe role is now scenario

File: extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/AutomationSnippet.java [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+            runners.put(runner.toJson());
+        }
+
+        return result;
+    }
+
+    @Override
+    public void shutdown() {
+    }
+}
```
Comment (AlanRosenthal) [inline extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/AutomationSnippet.java:289]: I think GitHub is saying this file is missing an empty line at the end

Comment (AlanRosenthal) [reply extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/AutomationSnippet.java:289]: here and elsewhere

File: extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/Pinger.kt [hunk 3]
```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Google LLC
```
Comment (AlanRosenthal) [inline extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/Pinger.kt:1]: 2024

**Review by AlanRosenthal — APPROVED** [review]

File: docs/mkdocs/src/apps_and_tools/bench.md (hunk 1 above)
Comment (barbibulle) [reply docs/mkdocs/src/apps_and_tools/bench.md]: done

File: extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/AutomationSnippet.java (hunk 2 above)
Comment (barbibulle) [reply extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/AutomationSnippet.java]: done

File: extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/Pinger.kt (hunk 3 above)
Comment (barbibulle) [reply extras/android/BtBench/app/src/main/java/com/github/google/bumble/btbench/Pinger.kt]: done

**Pull request merged**